src/scenarios/shared/const.py

```diff
@@ -14,3 +16,14 @@
 PUBDIR = 'pub'
 APPDIR = 'app'
 TRACEDIR = 'traces'
+TMPDIR = 'tmp'
+
+DOTNET = 'dotnet'
+PYTHON = 'py'
```

Comment:
You should pull this out of the environment. We set it into an environment variable called `Python`.

Reply:
Weird I didn't see the ```Python``` environment variable in the machine-setup.py dump? If that's already in the PATH we can just invoke ```py``` or ```python```. Or do you mean it should call ```%Python%```.